SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security
+
+The PrairieLearn team takes the security of our products and services seriously. Thanks for helping to make PrairieLearn secure for everyone.
+
+## Reporting a vulnerability
+
+If you believe you have found a vulnerability in any PrairieLearn software, please report it to us via coordinated disclosure. **Do not report suspected vulnerabilities publicly, including through GiHub issues or public Slack channels.** Instead, please send an email to security@prairielearn.com with as much relevant information as possible, including:
```

Comment:
```suggestion
If you believe you have found a vulnerability in any PrairieLearn software, please report it to us via coordinated disclosure. **Do not report suspected vulnerabilities publicly, including through GitHub issues or public Slack channels. You should assume that all Slack channels are public unless explicitly stated otherwise.** Instead, please send an email to security@prairielearn.com with as much relevant information as possible, including:
```

Reply:
I'll just discourage Slack in general:

```suggestion
If you believe you have found a vulnerability in any PrairieLearn software, please report it to us via coordinated disclosure. **Do not report suspected vulnerabilities publicly, including through GitHub issues or Slack.** Instead, please send an email to security@prairielearn.com with as much relevant information as possible, including:
```